README.md

```diff
@@ -34,3 +34,11 @@ make[1]: Leaving directory '.../linux-with-rust-support'
 For details about the Rust support, see https://github.com/Rust-for-Linux/linux.
 
 For details about out-of-tree modules, see https://www.kernel.org/doc/html/latest/kbuild/modules.html.
+
+## rust-analyzer
+
+Rust-for-Linux supports building a `rust-project.json` configuration for [`rust-analyzer`](https://rust-analyzer.github.io/), including for out-of-tree modules:
```

Comment:
I think you didn't notice the dashes -- GitHub did not highlight them because I edited more bits. It looks like if I only show that, it shows them:

```suggestion
Rust for Linux supports building a `rust-project.json` configuration for [`rust-analyzer`](https://rust-analyzer.github.io/), including for out-of-tree modules:
```

Reply:
Ah yes I did, thank you!